packages/runtime/container-runtime/src/opLifecycle/README.md

```diff
@@ -155,3 +171,98 @@ Notice that the sequence numbers don’t matter here, as all ops will be based o
 Additionally, as compression preserves the original uncompressed batch layout in terms of the number of ops by using empty ops to reserve the sequence numbers, this ensures that the clients will always receive the exact count of ops to rebuild the uncompressed batch sequentially.
 
 On the receiving end, the client will accumulate chunks 1 and 2 and keep them in memory. When chunk 3 is received, the original large, decompressed op will be rebuilt, and the runtime will then process the batch as if it is a compressed batch.
+
```

Comment:
@alexvy86 - FYI, since you asked about it on last meeting. Can you please help review it and help us make it better?
Thanks!